File: java/bypstest-api/src/byps/test/api/inherit/LemonService.java
package byps.test.api.inherit;

import byps.RemoteException;
import byps.test.api.auth.SessionInfo;

/**
 * 
 * @BSessionParamType com.wilutions.byps.test.api.auth.SessionInfo
 * 
 */
public interface LemonService extends FruitService {

  public String pick(SessionInfo sess, String fromTree) throws RemoteException;
  
}

File: csharp/bypstest-ser/src-ser/byps/test/api/inherit/LemonService.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using byps;

namespace byps.test.api.inherit
{
	
	// checkpoint byps.gen.cs.GenRemoteClass:93
	public interface LemonService : BRemote, 
		FruitService {
	
		String Pick(byps.test.api.auth.SessionInfo sess, String fromTree);
		void Pick(byps.test.api.auth.SessionInfo sess, String fromTree, BAsyncResult<String> asyncResult) ;
		Task<String> PickAsync(byps.test.api.auth.SessionInfo sess, String fromTree);
		
		
	}
}  // end namespace
